pkg/proxy/ipvs/proxier.go

```diff
@@ -1780,19 +1780,12 @@ func (proxier *Proxier) writeIptablesRules() {
 		"-j", "ACCEPT",
 	)
 
-	// The following two rules ensure the traffic after the initial packet
```

Comment:
This comment implies to me that at some point it was intentional to have two rules heres but we mistakenly made them the same.

Reply:
Interestingly in https://github.com/kubernetes/kubernetes/commit/8c6956e5bb4f9557e6ba459f1f6bbcb80e3b5b80#diff-d51765b83fe795b469e8a86276b12dc9L1517 we removed the cluster cidr check but didn't add a local detector check. 

@satyasm any idea why this is the case here?

Reply:
we didn't add the local detector check because the intent would not be the same. The original code seemed to be an optimization to accept previously accepted traffic if it is bound for the cluster. Since node CIDR does not represent the whole cluster, as discussed in the KEP (https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20191104-iptables-no-cluster-cidr.md#iptables---accepting-traffic-after-first-packet-after-being-accepted-by-kubernetes-rules), it was felt better to just remove it.